src/types/unix_time.ts

```diff
@@ -0,0 +1,27 @@
+export class UnixTime {
+	constructor(private readonly unixTime: number) {
+		if (this.unixTime < 0 || !Number.isInteger(this.unixTime) || !Number.isFinite(this.unixTime)) {
+			throw new Error('Unix time must be a positive integer!');
+		}
+	}
+
+	[Symbol.toPrimitive](hint?: string): number | string {
+		if (hint === 'string') {
+			this.toString();
+		}
+
+		return this.unixTime;
```

Comment:
For each of these `toPrimitive` we have to carefully think what are we gonna return for the `"default"` case
I think for most of the cases we are covered. Let's review this when all branches are mreged into the integration branch

Reply:
Bug on github: When you multiple-tap `add review commen` it adds as may as you tapped ahha